test/silicon_runners.jl

```diff
@@ -3,6 +3,8 @@ using DFTK
 
 include("testcases.jl")
 
+# Silicon RHF is a metal, so we add a bit of temperature to it
```

Comment:
Please do not abbreviate RHF = reduced HF, because for me RHF is restricted HF ... this messes with me each time.

Reply:
Ah, yes. This is very very annoying. I'd rename it to Hartree, but for some people Hartree means with self-interaction removed (the Hartree potential for orbital i uses the total density minus the density of orbital i). I'll make it explicit

Reply:
Names ... always difficult :smile: